.env.example

```
@@ -1,3 +1,4 @@
 REACT_APP_CHAIN_ENDPOINT=wss://peregrine.kilt.io
+UNIRESOLVER_ENDPOINT=https://dev-uniresolver.kilt.io
```

In a React context, env variables used in your app must begin with `REACT_APP_` to be included
```suggestion
REACT_APP_UNIRESOLVER_ENDPOINT=https://dev-uniresolver.kilt.io
```

---

Renamed